banzai_nres/background.py

```diff
@@ -0,0 +1,13 @@
+from banzai.stages import Stage
+import numpy as np
+import sep
+
+
+class BackgroundSubtractor(Stage):
```

Comment:
Background subtraction shouldn't be in this pull request as this request is focused on the extraction.